fud2/scripts/rtl_sim.rhai

```diff
@@ -28,9 +28,11 @@ fn sim_setup(e) {
     );
 
     // Rule for simulation execution.
+    // the stupid cat thing is just to pipe the contents of the log to stderr
+    // only when the sim exits with an error code
     e.rule(
         "sim-run",
-        "./$bin +DATA=$datadir +CYCLE_LIMIT=$cycle-limit $args > $out",
+        "./$bin +DATA=$datadir +CYCLE_LIMIT=$cycle-limit $args > $out || (cat $out >&2 && false)",
```

Comment:
I suppose this is because some Runt tests depend on actually seeing the output log from RTL simulation? That seems a little odd and maybe something we want to clean up, but probably not in the scope of this PR.

Reply:
Yes this specifically comes up because we want to see the error info in some tests, but also more closely replicates the original `fud` behavior as without this you would see that `verilator` aborted but would not see what actually caused it, which would be confusing. There's probably a better overall way of approaching this but I'm not sure what it is right now